lib/assert.js

```diff
@@ -286,6 +286,7 @@ class Assertions {
 			},
 			onSuccess: () => {
 				pass();
+				return true;
```

Comment:
Do we need to return `false` in `onError` or something?

Reply:
yes, we should.